cost/terminate_policy/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.5
+
+- remove permissions block and adding incident resource table
```

Comment:
There were no changes to permissions block.

Reply:
made appropriate changes.

Reply:
fixed .